### DISCUSSION TITLE
o1-preview model support

### DISCUSSION THREAD

**Discussion by wonrax** [CopilotC-Nvim/CopilotChat.nvim#416] [General]
GitHub recently announced that they now allow o1-preview model usage through Copilot[^1]. I tried setting the `model` to `o1-preview` in the config but it doesn't work, saying that the model is not supported.

[^1]: https://github.blog/news-insights/product-news/try-out-openai-o1-in-github-copilot-and-models/

**Comment by pidgeon777** [comment]
There is an open issue here:

https://github.com/CopilotC-Nvim/CopilotChat.nvim/issues/419
